_pkgdown.yml

```diff
@@ -3,6 +3,13 @@ url: http://r-lib.github.io/rcmdcheck
 development:
   mode: auto
 
+template:
+  package: tidytemplate
+  bootstrap: 5
+  includes:
+    in_header: |
+      <script defer data-domain="rcmdcheck.r-lib.org,all.tidyverse.org" src="https://plausible.io/js/plausible.js"></script>
```

Comment:
Not sure if this is needed here. Including it just for consistency. Can remove it if unnecessary.